Marsiple/Common/UI/MartianColors.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MartianColors.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 17/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+enum MatrianColor {
+    case red
+    case dark
+    case darkGray
+    case gray
+    case lightGray
+    
+    var value: UIColor {
+        switch self {
+        case .red:
+            return UIColor(ciColor: CIColor(red: 0.937, green: 0.251, blue: 0.349))
```

Comment:
We shouldn't use the `CIColor` to create colors - there's a more straightforward and simpler approach. 

Add a `UIColor` extension with a new convenience initialiser: 

```
public convenience init(red: Int, green: Int, blue: Int, alpha: CGFloat = 1) {
		self.init(red: CGFloat(red) / 255, green: CGFloat(green) / 255, blue: CGFloat(blue) / 255, alpha: alpha)
}
```